plugin/src/main/kotlin/com/jraska/module/graph/assertion/ModuleGraphAssertionsPlugin.kt

```diff
@@ -19,9 +19,8 @@ class ModuleGraphAssertionsPlugin : Plugin<Project> {
     GradleDependencyGraphFactory.create(evaluatedProject, configurationsToLook).serializableGraph()
   }
 
-  private val aliases by lazy {
```

Comment:
Hey, thanks for the open PR with proposed solution.

I think this might have fixed your issue, but I think it might be by pure chance and the root problem might be elsewhere in the way and order Gradle evaluates the project. The aliases should be deterministic and not depending on a way of time when being called - hence they are saved.

I will follow up on the issue you opened.

Reply:
Yeah, it did fixed. I compiled the plugin with these changes and applied the plugin using the jar and it is now considering the alias.

Reply:
Ok, please let's then follow up on the issue #238, because unless we understand why the problem is happening in your project, I cannot accept the PR. :)